### PR TITLE
LaTeX is LaTeX - and not LaTex

### PR DESCRIPTION
**Pull request by koppor**
Fix casing of "LaTeX" --> either "Latex" (code) or "LaTeX" (code comments, user-facing)

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Guys, don&#39;t trigger my OCD, please: it&#39;s LaTeX, not lATEX, lATex, lAtEx, lAteX, laTEx, laTeX, latEX, latex, LaTEX, LaTex, LatEx, LateX, LAtEX, LAtex, LATEx, LATeX, lATEx, lATeX, lAtEX, lAtex, laTEX, laTex, latEx, lateX, LaTEx, LatEX, Latex, LAtEx, LAteX, LATEX or LATex. <a href="https://twitter.com/hashtag/TeXLaTeX?src=hash&amp;ref_src=twsrc%5Etfw">#TeXLaTeX</a></p>&mdash; Paulo Cereda (@paulocereda) <a href="https://twitter.com/paulocereda/status/1229783564801265666?ref_src=twsrc%5Etfw">February 18, 2020</a></blockquote>  

In the Java code, one word should start with a capital letter followed by lower capital letters. See https://google.github.io/styleguide/javaguide.html#s5.3-camel-case for details. - I fixed that, too.

Regarding the user of JabRef, I changed it to "LaTeX"

### CODE REVIEW

File: src/main/java/org/jabref/gui/texparser/ParseTexDialogViewModel.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ public ParseTexDialogViewModel(BibDatabaseContext databaseContext, DialogService
         this.taskExecutor = taskExecutor;
         this.preferencesService = preferencesService;
         this.fileMonitor = fileMonitor;
-        this.texDirectory = new SimpleStringProperty(databaseContext.getMetaData().getLaTexFileDirectory(preferencesService.getUser())
+        this.texDirectory = new SimpleStringProperty(databaseContext.getMetaData().getLatexFileDirectory(preferencesService.getUser())
```
Comment (calixtus) [inline src/main/java/org/jabref/gui/texparser/ParseTexDialogViewModel.java]: Why not also 'this.latexDirectory' and ParseLatexDialogViewModel.java?

Comment (koppor) [reply src/main/java/org/jabref/gui/texparser/ParseTexDialogViewModel.java]: I will do that in a follow-up PR to keep things going.

**Comment by calixtus** [comment]
So let's merge... ?

**Comment by calixtus** [comment]
Localization test is failing...

Test keyValueShouldBeEqualForEnglishPropertiesMessages() FAILED
org.opentest4j.AssertionFailedError: expected: <LaTeX file directory=LaTeX file directory> but was: <LaTeX file directory=LaTex file directory>

**Comment by Siedlerchr** [comment]
Please no more l10n changes. We have a  couple of them at 100%

**Comment by koppor** [comment]
I kept them at 100% at https://github.com/JabRef/jabref/pull/6020/. Was not sooo hard to go through the languages and use Crowdin's translation memory.

**Pull request merged**